app/pages/Account/functions.js

```diff
@@ -0,0 +1,256 @@
+        )
+          return idx;
+        return null;
+      }, null);
+      if (inputIdx != null) {
+        // This reveal is for the current bid => Bid is revealed
+        isRevealed = true;
+        revealPrevOut = { hash: reveal.prevout.hash, index: inputIdx };
+        break;
+      }
+    }
+
+    // If bid not yet revealed and is in reveal period
+    if (!isRevealed) {
+      if (auction.state === "REVEAL") {
+        revealableHNS += blind;
+        revealableNum++;
+        if (
+          revealableBlock === null ||
+          auction.stats.blocksUntilClose < revealableBlock
+        ) {
+          revealableBlock = auction.stats.blocksUntilClose;
+        }
+      }
+      continue;
+    }
+
+    // At this point, the bid is revealed (at least)
+
+    // If still in reveal period, the blind is no longer locked
+    if (auction.state === "REVEAL") {
+      revealingHNS -= blind;
+      continue;
+    }
+
+    // Get the reveal coin from wallet
+    const revealCoin = await walletClient.getCoin(
+      revealPrevOut.hash,
+      revealPrevOut.index
+    );
+    console.log("revealCoin", auction.name, bid.value, revealCoin);
+
+    // Check if the reveal coin is the owner of the name
+    const revealCoinIsOwner =
+      auction.owner.hash === revealPrevOut.hash &&
+      auction.owner.index === revealPrevOut.index;
+    console.log("revealCoinIsOwner", revealCoinIsOwner);
+
+    // If the reveal coin is the owner, then the name is registerable
+    if (revealCoinIsOwner) {
+      registerableHNS += auction.highest - auction.value;
+      registerableNum++;
+      continue;
+    }
+
+    // If reveal coin exists, but not owner, then it is redeemable
+    if (revealCoin) {
+      // Is local?
+      if (revealCoin.height < auction.height) {
+        continue;
+      }
+      redeemableHNS += bid.value;
+      redeemableNum++;
+    }
+  }
+
+  // locked Bidding = bids in BIDDING state auctions
+  // locked Reveal = bids in REVEAL state auctions (includes blind if not revealed yet)
+  // locked Finished = bids that can be REDEEMed or REGISTERed
+  return {
+    lockedBalance: {
+      bidding: { HNS: biddingHNS, num: biddingNum },
+      revealing: { HNS: revealingHNS, num: revealingNum },
+      finished: {
+        HNS: registerableHNS + redeemableHNS,
+        num: registerableNum + redeemableNum,
+      },
+    },
+    actionableInfoBids: {
+      revealable: {
+        HNS: revealableHNS,
+        num: revealableNum,
+        block: revealableBlock,
+      },
+      redeemable: { HNS: redeemableHNS, num: redeemableNum },
+      registerable: { HNS: registerableHNS, num: registerableNum },
+    },
+  };
+}
+
+async function getStatsFromNames() {
+  let transferringDomains = new Set();
+  let transferringBlock = null;
+  let finalizableDomains = new Set();
+  let renewableDomains = new Set();
+  let renewableBlock = null;
+
+  const names = await walletClient.getNames();
+  console.log("names", names);
```

Comment:
clean up log statement

Reply:
Yeah, that's the last item on the todo list above. Keeping it for now as people review, helps find any numerical differences. Will remove before changing this PR from Draft to Ready to Merge.